core/src/avm2/activation.rs

```diff
@@ -1125,39 +1086,23 @@ impl<'a, 'gc> Activation<'a, 'gc> {
                 Op::Sxi8 => self.op_sxi8(),
                 Op::Sxi16 => self.op_sxi16(),
                 Op::Throw => self.op_throw(),
-                _ => self.unknown_op(op),
+                _ => unimplemented!("Encountered unimplemented AVM2 opcode {:?}", op),
```

Comment:
Proposal: make unimplemented dxns stuff Err at runtime like before; as for PushConstant(PushFloat, really), make it fail verification, just like it does in FP with the feature disabled. Alternatively, remove it entirely, so we throw `"Error #1011: Method contained illegal opcode."` like we already would for PushFloat4.

Reply:
The second approach won't work because some SWFs mention `Dxns`(`Late`) despite not actually using it.

Reply:
By „remove it entirely”, I only meant the PushConstant one.